e2e/test/scenarios/actions/actions-on-dashboards.cy.spec.js

```diff
@@ -916,20 +915,28 @@ const MODEL_NAME = "Test Action Model";
           });
 
           actionEditorModal().within(() => {
-            cy.findByTestId("native-query-editor").click();
-
-            cy.get(".ace_content").type(
-              _.times(23, () => "{leftArrow}")
-                .join("")
-                .concat("{backspace}{backspace}"),
-            );
-            cy.get(".ace_text-input").type(`{{ score }}`, {
-              force: true,
+            cy.get(".ace_content").click().type("{home}{shift+end}{backspace}");
+            const TEST_COLUMNS_QUERY = `UPDATE ${TEST_COLUMNS_TABLE} SET timestamp = {{ Timestamp }} WHERE id = {{ ID }}`;
+            cy.get(".ace_content").type(TEST_COLUMNS_QUERY, {
+              delay: 0,
               parseSpecialCharSequences: false,
             });
 
             cy.findByTestId("action-form-editor").within(() => {
-              cy.findAllByText("Number").click({ multiple: true });
+              cy.contains("ID")
+                .closest('[data-testid="form-field-container"]')
```

Comment:
I don't think targeting an element and trying to find its parent is the best way.

We should add a label or something similar to each section so we can go directly to a specific section.

Nothing needs to be done here, but I want you to be aware of that. This hasn't been discussed in FE guild sync yet, but it looks like a good candidate.

Reply:
I [created an RFC](https://www.notion.so/The-idiomatic-way-to-target-certain-sections-in-Cypress-dc2b11b8752a49fab04d4c7a2a285268) for this to be discussed in the next FE guild sync.